src/verifier.ts

```diff
@@ -172,4 +183,6 @@ export interface VerifierOptions extends SpawnArguments {
 	tags?: string[];
 	timeout?: number;
 	monkeypatch?: string;
+	format?: "json" | "RspecJunitFormatter";
```

Comment:
On a side note, we really don't want the name of the RspecJunitFormatter to leak into the javascript. The value of format should be "json" or nil, and if it is nil, the format option should not be included.

Reply:
Sure, makes sense. Not sure this level of restriction is needed anyway (plus it's only for people using Typescript)... all other params are only doing basic type checks, not enum-like checking

Reply:
Sorry, I remember what the RspecJunitFormatter does now. I thought it was the default one, but it's actually the XML formatter. I guess we can put it in there, but given that other options could be used in the future, I feel hesitant about putting the validation into this top level.

Reply:
Yep agreed. And this validation will only be for those using typescript anyway, rather than anyone. I'll remove it